resource/reapi/bindings/c/reapi_cli.cpp

```diff
@@ -91,33 +91,21 @@ extern "C" int reapi_cli_initialize (reapi_cli_ctx_t *ctx, const char *rgraph,
 }
 
 extern "C" int reapi_cli_match (reapi_cli_ctx_t *ctx,
-                                const char *match_op, const char *jobspec,
+                                enum match_op_t match_op, const char *jobspec,
```

Comment:
No need for `enum` type in function signature.
```suggestion
                               match_op_t match_op, const char *jobspec,
```